build/dependencies.props

```diff
@@ -13,4 +13,8 @@
     <XunitVersion>2.4.1</XunitVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-19270-01" PrivateAssets="All" />
```

Comment:
It's a bit weird to have package references in this file. Can't we have that in `packages.props` instead, or in the individual project files?

Reply:
Happy to move the version, but I'd rather the reference was here so it's everywhere for free, otherwise in the OAuth provider repo I'd have had to add it to 50+ files instead of one to achieve the same effect.

Reply:
K. Let's move it to `packages.props` so it keeps the versions file clean (and it doesn't apply to tests projects 😄 )